### ISSUE TITLE
16 bit support

### ISSUE THREAD

**Issue by polluks** [hippietrail/RetroGhidra#4]
FYI there's already some support
https://github.com/BartmanAbyss/ghidra-amiga
https://github.com/czietz/ghidraScripts_for_Atari
https://github.com/gnulnulf/Ghidra-TMS9900

**Comment by hippietrail** [comment]
Yep I think I've starred all of those.

I started work on an Amiga hunk loader and found out the first files I tried it on (from the oldest Fish Disk) broke every Hunk Parser on GitHub so I decided it must be hard to implement correctly.

I have stub loaders for Amiga and Atari binary executable formats that recognize them but don't load them yet.

I may have gotten further with the Sharp X68000 or Sinclair QL loaders. M68K is one of the CPUs I used to code in assembly back in the day.

**Comment by polluks** [comment]
I'll stay tuned.

**Comment by hippietrail** [comment]
I just pushed a working Sinclair QL loader and I pushed the working Atari ST loader a couple of days ago. The 68000-based Mac loader has been working for a long time. I'll do at least one of the Sharp X68000 executable formats next.

Are you using TI 99-4/A? What file formats are most popular?

Note that disk image formats are a lot harder and I'll be doing them as proper Ghidra FileSystems a bit later.  So for now I'm working with native executable formats, machine state snapshot formats, cartridge formats, and the like that just represent a single program.
